service_info/static/less/base/header/nav.less

```diff
@@ -51,14 +72,27 @@
       .align-items(center);
 
       &.collapsible-header {
+        &:not(.active) + .collapsible-body .overlay {
+          display: none;
```

Comment:
This is to fix a problem where a collapsible element in the process of being collapsed does not hide `position:absolute` child elements.